src/bos/server/controllers/v2/healthz.py

```diff
@@ -51,6 +51,7 @@ def get_v2_healthz():
 
     :rtype: Healthz
     """
+    LOGGER.debug("GET /v2/healthz invoked get_v2_healthz")
```

Comment:
You're doing a lot of enunciating what we're doing all over the place. And not just here, but in many places, it's just a restated of what function we're calling. Why? To what purpose? In this case, are we just curious how often we do a health check?

I assume the answer is simply what you wrote in the explanation, " The intention is to make it easier to trace execution to debug problems." It just seems annoying to have it spit out a reiteration of exactly what it's doing, but I'm guessing you will find it helpful in debugging things.

Reply:
In this case, I am just making these functions consistent with other ones which already had those statements (and not added by me). IMO we could remove those debug statements from all of the endpoints if desired, but I didn't want a situation where some had them and some did not, because that can lead to confusion.